### PR TITLE
add readme en doc

### PR DESCRIPTION
**Pull request by littletomatodonkey**
See title.

### CODE REVIEW

File: README_en.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+
+<div align="center">
+    <img src="./docs/images/models/V100_benchmark/v100.fp32.bs1.main_fps_top1_s.jpg" width="700">
+</div>
+
+The above figure shows some of the latest server-side pretrained models. It can be seen from the figure that when using V100 GPU with FP32 and TensorRT, the `Top1` accuracy of the ResNet50_vd_ssld pretrained model on ImageNet1k-val dataset is **82.4%** and that of ResNet101_vd_ssld pretrained model is 83.7%. These pretained models are obtained from  SSLD knowledge distillation solution provided by PaddleClas. The marks of the same color and symbol in the figure represent models of different model sizes in the same series. For the introduction of different models, FLOPS, Params and detailed GPU inference time (including the infecne speed of T4 GPU with different batch size), please refer to the documentation tutorial formal details: [https://paddleclas.readthedocs.io/zh_CN/latest/models/models_intro.html](https://paddleclas.readthedocs.io/zh_CN/latest/models/models_intro.html)
```
Comment (shippingwang) [inline README_en.md:24]: infecne -> inference

File: README_en.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+
+<div align="center">
+<img
+src="./docs/images/models/mobile_arm_top1.png" width="700">
+</div>
+
+
+The above figure shows performance of some commonly used mobile-side models, including MobileNetV1, MobileNetV2, MobileNetV3 and ShuffleNetV2 series. The inferece time is tested on Snapdragon 855 (SD855) with the batch size set as 1. the `Top1` accuracy of the MV3_large_x1_0_ssld, MV3_small_x1_0_ssld, MV1_ssld and MV2_ssld pretrained model on ImageNet1k-val dataset are 79%, 71.3%, 76.74%, 77.89%, respectively (M is short for MobileNet). MV3_large_x1_0_ssld_int8 is a quantizatied pretrained model for MV3_large_x1_0. More details about the mobile-side models can be seen in [**models introduction tutorial**](https://paddleclas.readthedocs.io/zh_CN/latest/models/models_intro.html)
```
Comment (shippingwang) [inline README_en.md:33]: performance -> the performance
inferece -> inference

File: README_en.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+<div align="center">
+<img
+src="./docs/images/image_aug/main_image_aug_s.jpg" width="600">
+</div>
+
+
+## Quick start
+
+Based on flowers102 dataset, you can easily experience different networks, pretrained models and SSLD knowledge distillation method in PaddleClas. More details can be seen in [**Quick start PaddleClas in 30 minutes**](https://paddleclas.readthedocs.io/zh_CN/latest/tutorials/quick_start.html).
```
Comment (shippingwang) [inline README_en.md:81]: 我觉得 不要用 you
可以用 one
one can easily

File: README_en.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+
+## Featured extension and application
+
+### A classification pretrained model with 100K categories
+
+The models trained on ImageNet1K dataset are often used as pretrained models for other classification tasks in practical applications due to lack of training data. However, there are only 1,000 categories in the ImageNet1K dataset, and the feature migration capability of the pretrained model is limited. Therefore, Baidu developed a tag system including 100,000 categories, with semantic information and different granularity. Through manual or semi-supervised methods, more than 55,000,000 training images have been collectet. The system is the largest image classification system and training set in the country and even the world. PaddleClas provides the ResNet50_vd model trained on this dataset. The following table shows the comparison of using the ImageNet pretrained model and the above 100,000 image classification pretrained model in some practical application scenarios. Using the 100,000 image classification pretrained model, the recognition accuracy can be increased by up to 30%.
```
Comment (shippingwang) [inline README_en.md:93]: collectet -> collected. 
in the country and even the world ->  It is the largest image classification system in China even in the worldwide


PaddleClas provides the ResNet50_vd model trained on this dataset. The following table shows the comparison of using the ImageNet pretrained model and the above 100,000 image classification pretrained model in some practical application scenarios. Using the 100,000 image classification pretrained model, the recognition accuracy can be increased by up to 30%.

File: README_en.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+
+### Object detection
+
+In recent years, object detection tasks atrtract a lot of attention in academia and industry. The ImageNet classification model is often used for pretrained model n object detection, which can directly affect the effect of object detection. Based on 82.39% ResNet50_vd pretrained model, PaddleDetection provides a Pratical Server-side Detection solution, PSS-DET. The solution contains many strategies that can effectively improve the performance while take limited extra computation cost, such as model pruning, better pretrained model, deformable convolution, cascade rcnn, autoaugment, libra sampling and multi-scale training. Compared with the 79.12% ImageNet1k pretrained model, the 82.39% model can help improve the COCO mAP by 1.5% without any computation cost. Using PSS-DET, the inference speed on single V100 GPU can reach 20FPS when COCO mAP is 47.8%, and reach 61FPS when COCO mAP is 41.6%. For more details, please refer to [**Object Detection tutorial**](https://paddleclas.readthedocs.io/zh_CN/latest/application/object_detection.html).
```
Comment (shippingwang) [inline README_en.md:110]: atrtract -> attract
n -> in 
Pratical -> practical 
while take -> taking

File: README_en.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
+# PaddleClas
+
+**Book**: https://paddleclas.readthedocs.io
+
+**Quick start PaddleClas in 30 minutes**: https://paddleclas.readthedocs.io/zh_CN/latest/tutorials/quick_start.html
+
+## Introduction
+
+PaddleClas is a tool set for image classification tasks prepared for the industry and academia. It helps users train better computer vision models and apply in real scenarios.
```
[the remark this answers is not in the document]
Comment (littletomatodonkey) [reply README_en.md]: fixed

File: README_en.md [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+
+<div align="center">
+    <img src="./docs/images/main_features_s_en.png" width="700">
+</div>
+
+## Rich model zoo
+
+Based on ImageNet1k dataset, PaddleClas provides a brief introduction to 23 series of imaget classification networks such as ResNet, ResNet_vd, Res2Net, HRNet, and MobileNetV3 besides their reproduction configurations and training techniques. At the same time, the corresponding 117 image classification pretrained models are also open source. The GPU inference time of the server-side model is evaluated based on TensorRT. The CPU inference time and the mobile-side model storage size are evaluated on the Snapdragon 855 (SD855). For more detailed information of the supported pretrained models and their download links, please refer to [**models introduction tutorial**](https://paddleclas.readthedocs.io/zh_CN/latest/models/models_intro.html).
```
[the remark this answers is not in the document]
Comment (littletomatodonkey) [reply README_en.md]: fixed

File: README_en.md (hunk 1 above)
Comment (littletomatodonkey) [reply README_en.md]: fixed

File: README_en.md (hunk 2 above)
Comment (littletomatodonkey) [reply README_en.md]: fixed

File: README_en.md (hunk 3 above)
Comment (littletomatodonkey) [reply README_en.md]: done

File: README_en.md (hunk 4 above)
Comment (littletomatodonkey) [reply README_en.md]: fixed

File: README_en.md (hunk 5 above)
Comment (littletomatodonkey) [reply README_en.md]: fixed

File: README_en.md [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+- [ ] Reproduction and performance evaluation of EfficientLite, GhostNet and RegNet.
+
+
+## High-level support
+
+In addition to providing rich classification network structures and pretrained models, PaddleClas also supports a series of algorithms or tools that help to improve the effectiveness and efficiency of image classification tasks.
+
+### knowledge distillation (SSLD)
+
+Knowledge distillation refers to using the teacher model to guide the student model to learn specific tasks, ensuring that the small model has a relatively large effect improvement with the computation cost unchanged, and even obtains similar accuracy with the large model. PaddleClas provides a Simple Semi-supervised Label Distillation method (SSLD). With this method, different models on ImageNet1k-val dataset has 3% absolute improvement(Top1 accuracy) on ImageNet1k-val dataset. The following figure shows the models' performance after SSLD.
```
Comment (WuHaobo) [inline README_en.md:45]: “models on ImageNet1k-val dataset has”
should be "models .... have" or "model ....has" ?

Comment (littletomatodonkey) [reply README_en.md]: fixed

File: README_en.md [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+
+PaddleClas is a toolset for image classification tasks prepared for the industry and academia. It helps users train better computer vision models and apply them in real scenarios.
+
+<div align="center">
+    <img src="./docs/images/main_features_s_en.png" width="700">
+</div>
+
+## Rich model zoo
+
+Based on ImageNet1k dataset, PaddleClas provides a brief introduction to 23 series of image classification networks such as ResNet, ResNet_vd, Res2Net, HRNet, and MobileNetV3 besides their reproduction configurations and training techniques. At the same time, the corresponding 117 image classification pretrained models are also open source. The GPU inference time of the server-side model is evaluated based on TensorRT. The CPU inference time and the mobile-side model storage size are evaluated on the Snapdragon 855 (SD855). For more detailed information on the supported pretrained models and their download links, please refer to [**models introduction tutorial**](https://paddleclas.readthedocs.io/zh_CN/latest/models/models_intro.html).
```
Comment (dyning) [inline README_en.md:17]: besides 的语法不太通

Comment (dyning) [inline README_en.md:17]: 单复数

Comment (dyning) [reply README_en.md:17]: 链接是不是先不放

File: README_en.md [hunk 10]
```diff
@@ -0,0 +1,151 @@
...
+
+
+<div align="center">
+<img
+src="./docs/images/image_aug/image_aug_samples_s_en.jpg" width="800">
+</div>
+
+
+PaddleClas provides the reproduction of the above 8 data augmentation algorithms and the evaluation of the effect in a unified environment. The following figure shows the performance of different data augmentation methods based on ResNet50. Compared with the standard transformation, using data augmentation, the recognition accuracy  can be increased by up to 1%. For more detailed introduction of data augmentation methods, please refer to the [**data augmentation tutorial**](https://paddleclas.readthedocs.io/zh_CN/latest/advanced_tutorials/image_augmentation/index.html).
```
Comment (WuHaobo) [inline README_en.md:71]: "recognition accuracy  can be" 是不是多了一个空格

**Pull request merged**